free-1.md

```diff
@@ -58,11 +117,25 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+About 30-40% of the time would be spent engaging with the language aspects, while 60-70% will be working on the systems aspects of this project. 
+
+Less time is spent on language aspects because there are conventions to how subtitles should look on a video. Nonetheless, some language decisions will need to be made regarding annotation, as these types of notes have slightly different conventions between different types of media (ex. American television vs Japanese anime).
+
+The systems aspects of this project are more complicated. While inputs and outputs are easily definable, the inner workings of parsings may be difficult because of large numbers of potential edge cases.
+
+
 ### Scope
 
 _How big an idea is this? How ambitious is this project?_
 
+I think this is a relatively big idea, but modular enough that it can easily be scoped down. Particular avenues to make this project easier are establishing fixed formats of inputs/outputs and not creating the visual editor.
+
 ### Benefits and drawbacks
 
 _Why might this be a good idea for a project? Why might this not be a good idea
 project?_
+
+This might be a good idea for a project since it essentially boils down to text processing. String parsing is something I have more experience with than parsing other types of media. The line-by-line nature of subtitling means that there are already some sets of rules and constraints, giving me a template to convert those into code logic. This project is also independent of any one specific programming language, meaning more freedom to choose one that has optimal functions to do the task at hand. 
+
+Drawbacks to this project include the lack of an existing DSL in this domain. I would have to be careful about language design decisions because there is no reference for sanity checking. This also makes it harder to identify drawbacks within the implementation. Another potential downfall is that for this DSL to be useful, I should implement some sort of visualization like ContextFree for proofreading subtitles. However, creating that visualization might become the biggest risk of doing this project.
```

Comment:
I agree with the visualization point. However, you clearly know a lot about this niche and seem enthusiastic about the topic idea. I would totally go after it if you think it is achievable